13-Tenant Security Solution Setup/Scripts/RemediationScripts/Remediate-DontUseNonADIdentities.ps1

```diff
@@ -0,0 +1,364 @@
+function Pre_requisites
+{
+    <#
+    .SYNOPSIS
+    This command would check pre requisities modules.
+    .DESCRIPTION
+    This command would check pre requisities modules to perform remediation.
+	#>
+
+    Write-Host "Required modules are: Az.Resources, Az.Account" -ForegroundColor Cyan
+    Write-Host "Checking for required modules..."
+    $availableModules = $(Get-Module -ListAvailable Az.Resources, AzureAD, Az.Accounts)
```

Comment:
Do we need to check AzureAD module or not?

Reply:
No, because we are using ARM API to get eligible PIM role assignment and active role assignment we are getting from "Get-AzRoleAssignment" itself.